tests/std/tests/P0896R4_ranges_test_machinery/test.compile.pass.cpp

```diff
@@ -42,13 +42,29 @@ constexpr bool iter_test() {
     STATIC_ASSERT(!to_bool(Eq) || sentinel_for<I, I>);
     STATIC_ASSERT(!to_bool(Eq) || !to_bool(Diff) || sized_sentinel_for<I, I>);
 
-    if constexpr (to_bool(Wrapped)) {
+    if constexpr (Wrapped == WrappedState::ignorant) {
+        STATIC_ASSERT(sentinel_for<S, iterator<Category, Element, Diff, Eq, Proxy, WrappedState::wrapped>>);
+        STATIC_ASSERT(!sentinel_for<S, iterator<Category, Element, Diff, Eq, Proxy, WrappedState::unwrapped>>);
+
+        STATIC_ASSERT(sentinel_for<sentinel<Element, WrappedState::wrapped>, I>);
+        STATIC_ASSERT(!sentinel_for<sentinel<Element, WrappedState::unwrapped>, I>);
+
+        STATIC_ASSERT(!_Unwrappable_v<I>);
+        STATIC_ASSERT(!_Unwrappable_v<S>)
+    } else if constexpr (Wrapped == WrappedState::wrapped) {

```

Comment:
This test is failing due to a missed semicolon:
```
C:\a\1\s\tests\std\tests\P0896R4_ranges_test_machinery\test.compile.pass.cpp(54): error C3878: syntax error: unexpected token 'else' following 'statement_seq'
```
Please always run all added/modified tests before pushing changes. `git diff --name-only main @` can be used to find which files have changed (alter appropriately if you haven't merged the latest `main`).